bucket/librewolf.json

```
@@ -22,6 +21,10 @@
     "checkver": {
         "url": "https://gitlab.com/api/v4/projects/13852981/releases",
         "jsonpath": "$[0].assets.links[1].url",
-        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-([\\d.]+)\\.en-US\\.win64\\.zip"
+        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-(?<version>[\\d.]+)\\.en-US\\.win64\\.zip"
```

```suggestion
        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-([\\d.]+)\\.en-US\\.win64\\.zip"
```

Version is automatically captured, so no need to name it

---

ah, I saw them capture version in the openjdk example and assumed it was necessary